src/store/accounts/nonce.ts

```diff
@@ -20,11 +22,27 @@ export const latestNonceState = atom(get => {
   return get(localNoncesState([network.url, address || '']));
 });
 
+// TODO use AccountsApi.getAccountNonces instead (not yet available in @stacks/blockchain-api-client)
```

Comment:
this should be currently available, if not bump the client lib version to the -beta version

Reply:
It’s there only on the develop branch

So I used GitHub as source in package.json with

    "@stacks/blockchain-api-client": "blockstack/stacks-blockchain-api#develop”,

This resolves well for the getAccountNonces but I am facing some weird resolution error when I run yarn dev

Like

Module not found: Error: Can't resolve '@stacks/blockchain-api-client' in ‘./src/store/assets'
 @ ./src/store/assets/tokens.ts 13:0-54 63:24-39
 @ ./src/common/hooks/use-assets.ts 9:0-136 15:22-33 25:22-45 36:11-30 49:11-33 62:11-24
 @ ./src/pages/send-tokens/send-tokens.tsx 29:0-65 49:15-36 138:73-94
 @ ./src/routes.tsx 16:0-64 103:40-54
 @ ./src/app.tsx 14:0-34 27:260-266
 @ ./src/index.tsx 5:0-24 8:54-57

I think we should wait for the official release.

Reply:
@beguene in the future, this is not the best way to use the latest version `"@stacks/blockchain-api-client": "blockstack/stacks-blockchain-api#develop”,`, you can do `yarn add @stacks/blockchain-api-client@beta` which has this method

Reply:
the package has been updated, however, so the latest version supports this now.

Reply:
Sure, this should work now !

Reply:
I used the api now thanks to the new release !